tsconfig.json

```diff
@@ -33,7 +33,9 @@
       {
         "name": "typescript-strict-plugin",
         "paths": [
+          "./packages/@react-spectrum/action",
```

Comment:
Shouldn't this be actionbar and actiongroup?

Reply:
it'll match both